core-java/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -255,12 +269,21 @@ private void apply(Message event) throws InvocationTargetException {
             restore((Snapshot) event);
             return;
         }
-
         invokeApplier(event);
     }
 
-    private void putUncommitted(EventRecord record) {
-        uncommittedEvents.add(record);
+    /**
+     * Returns a set of classes of state-neutral events.
+     *
+     * <p>An event is state-neutral if there is no need to modify a state of an aggregate on this event.
+     * It is not required to create applier methods for these events.
```

Comment:
I would say: "Instead of creating empty applier methods for such events, override this method returning immutable set of event classes. `<Code example>`"